File: filedata/entity/c#/Baseaction.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 系统权限表
    /// </summary>
    [Table("Base_Action")]
    public class Baseaction
    {
        
        /// <summary>
        /// 主键
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 创建时间
        /// </summary>
        [Column("CreateTime")]
        public DateTime CreateTime { get; set; }
    
    
        /// <summary>
        /// 创建人Id
        /// </summary>
        [Column("CreatorId")]
        public string CreatorId { get; set; }
    
    
        /// <summary>
        /// 否已删除
        /// </summary>
        [Column("Deleted")]
        public int Deleted { get; set; }
    
    
        /// <summary>
        /// 父级Id
        /// </summary>
        [Column("ParentId")]
        public string ParentId { get; set; }
    
    
        /// <summary>
        /// 类型,菜单=0,页面=1,权限=2
        /// </summary>
        [Column("Type")]
        public int Type { get; set; }
    
    
        /// <summary>
        /// 权限名/菜单名
        /// </summary>
        [Column("Name")]
        public string Name { get; set; }
    
    
        /// <summary>
        /// 菜单地址
        /// </summary>
        [Column("Url")]
        public text Url { get; set; }
    
    
        /// <summary>
        /// 权限值
        /// </summary>
        [Column("Value")]
        public string Value { get; set; }
    
    
        /// <summary>
        /// 是否需要权限(仅页面有效)
        /// </summary>
        [Column("NeedAction")]
        public int NeedAction { get; set; }
    
    
        /// <summary>
        /// 图标
        /// </summary>
        [Column("Icon")]
        public string Icon { get; set; }
    
    
        /// <summary>
        /// 排序
        /// </summary>
        [Column("Sort")]
        public int Sort { get; set; }
    
    
    }
}

File: filedata/entity/java/BaseAction.java

package entity;
// 系统权限表
public class Baseaction  extends BaseDomain{
    public Baseaction() {

    }
    
    //系统权限表
    private String Id;
        
    //系统权限表
    private Date CreateTime;
        
    //系统权限表
    private String CreatorId;
        
    //系统权限表
    private tinyint(4) Deleted;
        
    //系统权限表
    private String ParentId;
        
    //系统权限表
    private int(11) Type;
        
    //系统权限表
    private String Name;
        
    //系统权限表
    private text Url;
        
    //系统权限表
    private String Value;
        
    //系统权限表
    private tinyint(4) NeedAction;
        
    //系统权限表
    private String Icon;
        
    //系统权限表
    private int(11) Sort;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public Date getCreatetime() {
        return Createtime;
    }
    public void setCreatetime(Date Createtime) {
        this.Createtime = Createtime;
    }
            
    public String getCreatorid() {
        return Creatorid;
    }
    public void setCreatorid(String Creatorid) {
        this.Creatorid = Creatorid;
    }
            
    public tinyint(4) getDeleted() {
        return Deleted;
    }
    public void setDeleted(tinyint(4) Deleted) {
        this.Deleted = Deleted;
    }
            
    public String getParentid() {
        return Parentid;
    }
    public void setParentid(String Parentid) {
        this.Parentid = Parentid;
    }
            
    public int(11) getType() {
        return Type;
    }
    public void setType(int(11) Type) {
        this.Type = Type;
    }
            
    public String getName() {
        return Name;
    }
    public void setName(String Name) {
        this.Name = Name;
    }
            
    public text getUrl() {
        return Url;
    }
    public void setUrl(text Url) {
        this.Url = Url;
    }
            
    public String getValue() {
        return Value;
    }
    public void setValue(String Value) {
        this.Value = Value;
    }
            
    public tinyint(4) getNeedaction() {
        return Needaction;
    }
    public void setNeedaction(tinyint(4) Needaction) {
        this.Needaction = Needaction;
    }
            
    public String getIcon() {
        return Icon;
    }
    public void setIcon(String Icon) {
        this.Icon = Icon;
    }
            
    public int(11) getSort() {
        return Sort;
    }
    public void setSort(int(11) Sort) {
        this.Sort = Sort;
    }
             
    @Override
    public String toString() {
        return "Base_Action[Id=" + Id + ",CreateTime=" + CreateTime + ",CreatorId=" + CreatorId + ",Deleted=" + Deleted + ",ParentId=" + ParentId + ",Type=" + Type + ",Name=" + Name + ",Url=" + Url + ",Value=" + Value + ",NeedAction=" + NeedAction + ",Icon=" + Icon + ",Sort=" + Sort + "]";
    }
        
    
}
        